### PR TITLE
Bump version to 0.15.8 and update CHANGELOG

### PR DESCRIPTION
**Pull request by systemcatch**
I've updated as many links as possible since the redirects don't appear to be working? :thinking:

### CODE REVIEW

**Comment by jadchaar** [comment]
Hey @systemcatch I fixed a bunch of remaining references on this branch: https://github.com/arrow-py/arrow/compare/systemcatch-changelog-0158. I was going to replace this PR with it, but I thought it is best for you to keep ownership! I could not figure out how to push directly to this PR.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+0.15.8 (2020-07-19)
```
Comment (jadchaar) [inline CHANGELOG.rst:4]: Update the date :).

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Changelog
```
Comment (jadchaar) [inline CHANGELOG.rst:1]: Do we want to mention Python 2 and 3.5 deprecation warnings in the changelog? And warn about the impending drop? Eg. 0.15.x or 0.16.x are the final releases for Python 2.7 and 3.5?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
 =========
 
+0.15.8 (2020-07-19)
+-------------------
+
+- [NEW] Added ``humanize`` week granularity translation for Czech.
+- [FIX] ``arrow.get`` will now parse weekdays with no other inputs, see `#446 <https://github.com/arrow-py/arrow/issues/446>`_.
```
Comment (jadchaar) [inline CHANGELOG.rst:8]: I would change this to: "`arrow.get` will now pick sane defaults when weekdays are passed with particular token combinations".

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,6 +1,14 @@
...
+- [INTERNAL] Moved arrow into an `organization <https://github.com/arrow-py>`_.
```
Comment (jadchaar) [inline CHANGELOG.rst:9]: I would change this to: "Moved arrow to an organization. The repo can now be found `here <https://github.com/arrow-py/arrow>`_."

**Comment by systemcatch** [comment]
> Hey systemcatch I fixed a bunch of remaining references on this branch: https://github.com/arrow-py/arrow/compare/systemcatch-changelog-0158. I was going to replace this PR with it, but I thought it is best for you to keep ownership! I could not figure out how to push directly to this PR.

Hey @jadchaar I guess it would have been easier for me to make a branch on the main repo.

For reference assuming you have my remote added you can push to the branch like so;

```shell
git checkout -b changelog-0158 systemcatch/changelog-0158
# make changes
git push systemcatch changelog-0158
```

I'll make the changes suggested but let's not merge this until @krisfremen gets the warning setup.

**Pull request merged**